src/layout/Header/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import './style.css';
+
+const Header = () => {
+    return (
+        <header>
+            <span id="logo"><i className="fab fa-github"></i></span>
```

Comment:
Love the use of FontAwesome, so easy to bring in and makes an app look so much better!